tensorflow/core/common_runtime/direct_session_with_tracking_alloc_test.cc

```diff
@@ -138,7 +138,8 @@ TEST(DirectSessionWithTrackingAllocTest, CostModelWarmup) {
   DirectSession* ds = static_cast<DirectSession*>(session.get());
   CostModelManager::CostModelMap cost_models;
   ds->ExportCostModels(&cost_models);
-  CHECK_EQ(cost_models.size(), 1);
+  ASSERT_GE(2, cost_models.size());
```

Comment:
Fixed in: https://github.com/tensorflow/tensorflow/pull/9117/commits/73e405572a59ba8ab1d92dc3f7cddff5c9fb240c 

That should not be there. Auto merge conflict resolution cannot be trusted.